### PR TITLE
Adding repair to all recipes, adjusting existing

### PR DESCRIPTION
**Pull request by ProgrammerDan**
matching agreed on balance spreadsheet, including updates column.

Dowsing is missing and Emerald Extractor are missing.

### CODE REVIEW

**Comment by ProgrammerDan** [comment]
@Maxopoly If this looks good just merge, thanks.

File: configCivcraft.yml [hunk 1]
```diff
@@ -1267,92 +1270,81 @@ factories:
...
-         amount: 5
-    health_gained: 65
+      eye of ender:
+        material: EYE_OF_ENDER 
+        lore:
+         - Essence
+        enchants:
+          dura:
+            enchant: DURABILITY
+        amount: 1
```
Comment (Maxopoly) [inline configCivcraft.yml]: This doesnt seem right

Comment (ProgrammerDan) [inline configCivcraft.yml]: fixed

**Comment by Maxopoly** [comment]
Did you go through all factories to assign the right recipe to them? Most of those were probably wrong

**Comment by ProgrammerDan** [comment]
Yeah I went through every recipe. Probably worth a second eye. I'll PM you the burndown list I followed -- just a copy of the one we were using.

**Comment by ttk2** [comment]
drop this in templates when its ready.

Good work guys.

**Comment by ProgrammerDan** [comment]
Not seeing any new objections or corrections... merging.

**Pull request merged**